packages/build-essentials/src/config/transitionConfig.js

```diff
@@ -0,0 +1,35 @@
+const upperFirst = require('lodash.upperfirst');
+
+const config = {
+    transition: {
+        fast: '.1s',
+        default: '.2s',
+        medium: '.3s',
```

Comment:
Btw, these kind of refactoring really raise the UX-related questions, e.g. is `.2s` really distinguishable from `.3s`?
But this should obviously be taken care of by the UX guild and not here.

Reply:
To be honest I see no difference ... in my point of view we could maybe drop 200ms and use 300ms as default... then we have just slow, fast and default.

Reply:
Or 250 as default and 100 as fast and 500 as slow.

Reply:
Agreed.

Reply:
Adjusted that now.